spec/models/timeline/monitoring_review_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe Timeline::MonitoringEvent do
+  let(:date) { 1.day.ago.strftime("%F") }
+  let(:review_id) { "Test-Review-ID" }
+  let(:issue_id) { "Test-Issue-ID" }
+  # just setting the start_date manually because we aren't triggering the before_validation callback
+  let(:event_param) { IssueMonitoringReview.new review_id: review_id, issue_id: issue_id, start_date: date }
+  let(:monitoring_review) { event_param.monitoring_review }
+  subject { Timeline::MonitoringEvent.new(event_param) }
+
+  describe "#init" do
+    it "assigns attr_readers inherited from Timeline::Event" do
+      expect(subject.name).to eq review_id
+      expect(subject.date).to eq Date.parse(date)
+    end
+  end
+
+  describe "#review_link" do
+    it "returns a link that opens in a new tab" do
+      expect(subject.review_link).to match /\A<a .*>#{review_id}<\/a>\z/
+      expect(subject.review_link).to include 'target="_blank"'
+    end
+
+    context "authorization error loading Monitoring API" do
+      xit "returns just the review ID with no link"
+    end
+  end
+
+  describe "#review_type" do
+    it "returns the review type" do
+      expect(subject.review_type).to be_a(String)
```

Comment:
[suggestion] can we have it match on the options? 
`expect(subject.review_type.include?(FfakerWrapper::REVIEW_TYPE)).to be true` (or something like that?)